examples/pipelineruns/conditional-pipelinerun.yaml

```diff
@@ -28,22 +28,33 @@ spec:
 apiVersion: tekton.dev/v1alpha1
 kind: Task
 metadata:
-  name: list-files
+  name: create-readme-file
 spec:
-  inputs:
+  outputs:
```

Comment:
oh, I though we deprecated `git` pipeline resource as output, at least until we have an actual output resource

Reply:
Ahh, did not know that....I can switch the example to a GCS resource or something